index.html

```diff
@@ -188,5 +188,23 @@ <h2 class="presentation2">About <br> myself <br></h2>
 </div>
 </section>
 <!-- about myself finish-->
+  <!-- contact form start -->
+<section class="sec3">
+      <div class="contact">
+          <div class="contact_text">
+          <h2 class="contactme">Contact me</h2>
+          <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          </div>
+          <form>
```

Comment:
- [ ] You should submit form data to `Formspree` as per the requirement for the project that says `Collect data by using the `[Formspree service](https://formspree.io/html). You can find an example of how you can use Formspree if you click the provided here or in you activity.
- [ ] You should use Flexbox to layout form elements. `In order to lay out the elements on the page you should use Flexbox`